.github/workflows/test_security.yml

```diff
@@ -0,0 +1,96 @@
+name: Security test workflow for k-NN
+on:
+  push:
+    branches:
+      - "*"
+      - "feature/**"
+  pull_request:
+    branches:
+      - "*"
+      - "feature/**"
+
+jobs:
+  Build-ad:
+    strategy:
+      matrix:
+        java: [ 11,17 ]
+        os: [ubuntu-latest]
+      fail-fast: false
```

Comment:
why this is false? I think we should make it true.

Reply:
ack